p2p/dht/bootstrap.go

```diff
@@ -86,8 +85,12 @@ loop:
 			if gotpeers || tries >= bootstrapTries {
 				// TODO: consider choosing a random key that is close to the local id
 				// or TODO: implement real kademlia refreshes - #241
-				rnd, _ := crypto.GetRandomBytes(32)
-				searchFor = base58.Encode(rnd)
+				var err error
+				_, searchFor, err = cryptoBox.GenerateKeyPair()
```

Comment:
consider creating a public key from random bytes. might be cheaper.

Reply:
done

Reply:
done